core/finality-grandpa/src/communication/mod.rs

```diff
@@ -303,7 +303,7 @@ impl<B: BlockT, N: Network<B>> NetworkBridge<B, N> {
 							msg.message.auth_data.iter().map(move |(_, a)| {
 								format!("{}", a)
 							}).collect();
-						telemetry!(CONSENSUS_INFO; "afg.received_commit";
+						telemetry!(CONSENSUS_DEBUG; "afg.received_commit";
```

Comment:
could get noisy.

Reply:
Since we don't have a use case now I removed it.